src/php/lib/Grpc/AbstractCall.php

```diff
@@ -114,9 +114,7 @@ public function cancel()
     protected function _serializeMessage($data)
     {
         // Proto3 implementation
-        if (method_exists($data, 'encode')) {
-            return $data->encode();
-        } elseif (method_exists($data, 'serializeToString')) {
+       if (method_exists($data, 'serializeToString')) {
```

Comment:
I think we can remove this `if` clause too.

Reply:
ok